R/internal.R

```diff
@@ -527,6 +527,14 @@ omitcoef <- function(m, omit.coef) {
   return(m)
 }
 
+# function to select, omit, reorder, and rename coefficients
+custommap <- function(m, custom.coef.map) {
+    variables <- names(custom.coef.map)
+    variables <- variables[variables %in% row.names(m)]
+    out <- m[variables, ]
+    row.names(out) <- custom.coef.map[variables]
+    return(out)
+}
```

Comment:
This function seems to fail when the user provides only one element, as in `c("test", "new coefname")`.